frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -110,12 +110,12 @@ export const VMDetailsList: React.FC<VMResourceListProps> = ({
         <VMStatuses vm={vm} vmi={vmi} pods={pods} migrations={migrations} />
       </VMDetailsItem>
 
-      <VMDetailsItem title="Pod" idValue={prefixedID(id, 'pod')} isNotAvail={!launcherPod}>
-        {launcherPod && (
+      <VMDetailsItem title="VM Instance" idValue={prefixedID(id, 'vmi')} isNotAvail={!launcherPod}>
```

Comment:
why are we discarding the launcher pod?

Reply:
hmm, the idea was that VM is the owner of VMI and VMI is the owner of Pod, so in the VM view we only need to show the VMI ...

On the other hand, I do not think it's bad to show the pod ... @matthewcarleton WDYT ?

Reply:
re-added the pod

Reply:
+1

not sure if it should be here or in the VMI, but currently you cannot get to the pod through the VMI - and consequently not see the logs/events